Frontend/src/pages/dashboard/FarmerDash.jsx

```
@@ -741,18 +582,10 @@ function StatisticsDashboard() {
               const height = (item.amount / 10000) * 100;
               return (
-                <div
-                  key={item.month}
-                  className="flex flex-col items-center gap-2"
-                >
-                  <div
-                    className="w-16 bg-primary rounded-t-md"
-                    style={{ height: `${height}%` }}
-                  ></div>
+                <div key={item.month} className="flex flex-col items-center gap-2">
+                  <div className="w-16 bg-primary rounded-t-md" style={{ height: `${height}%` }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)